sdks/wasm/framework/src/WebAssembly.Net.Http/WasmHttpMessageHandler.cs

```diff
@@ -168,8 +146,10 @@ private async Task doFetch (TaskCompletionSource<HttpResponseMessage> tcs, HttpR
 
 				HttpResponseMessage httpresponse = new HttpResponseMessage ((HttpStatusCode)Enum.Parse (typeof (HttpStatusCode), status.Status.ToString ()));
 
-				httpresponse.Content = StreamingSupported && StreamingEnabled
-				    ? new StreamContent (wasmHttpReadStream = new WasmHttpReadStream (status))
+				var streamingEnabled = request.Properties.TryGetValue ("StreamingEnabled", out var streamingEnabledValue) && (bool)streamingEnabledValue;
```

Comment:
Is the exception generated going to be sensical here if StreamingEnabled is ```1``` or ```"true"``` instead of ```true```?

Reply:
Right now you get a vanilla TypeCastException. I can have this throw a more detailed error.